adm/systemd/beanstalkd.service

```diff
@@ -3,4 +3,4 @@ Description=Beanstalkd is a simple, fast work queue
 
 [Service]
 User=nobody
-ExecStart=/usr/bin/beanstalkd
+ExecStart=/usr/local/bin/beanstalkd
```

Comment:
Why does the path need to change? And why not used the $(PREFIX) in the Makefile, and add a `sed` command?

I.e. you could change this line to:

```
ExecStart=@PREFIX@/beanstalkd
```

and then have in the Makefile:

```
sed -i -e 's/@PREFIX@/$(PREFIX)/g' ...
```

when the file gets installed.